### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by sebwolf-de**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Which version do you use? Provide branch and commit id.
+2. Which build settings do you use? 
+3. On which machine does your problem occur?
```
Comment (krenzland) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Also: compiler (version), module list (if on clusters)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+4. Provide parameter/material files.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (krenzland) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Also: actual behavior?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (krenzland) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe add a default label. Or is this not only for bugs?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (sebwolf-de) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Actual behaviour should be given in "describe the bug"
But I'll add more text to make it unambiguous.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (sebwolf-de) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Default label will be bug. 
AFAIK people can still create issues freely e.g. feature requests,...

**Pull request merged**